app/assets/stylesheets/about.scss

```diff
@@ -0,0 +1,116 @@
+// Place all the styles related to the aboutme controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+.name {
+  display: inline-flex;
+  justify-content: center;
+  background-color: #55a8c9;
+  flex-wrap: nowrap;
+  width: 100%;
+  h1 {
+    font-size: 150px;
+    text-align: center;
+    font-family: fantasy;
+    padding-top: 50px;
+    position: relative;
+    padding: 0px 150px;
+  }
+  img {
+    padding-right: 0;
+    margin-right: 0;
+    padding-top: 75px;
+    width: 300px;
+    height: 300px;
+  }
+}
+
+.padding {
```

Comment:
The classname padding here is a bit misleading as it's setting the padding and the margin to 0 - how about something like `.thin` or `.narrow`?

Reply:
Thats a good point, didn't think of that, I'll change it to .thin